### DISCUSSION TITLE
How can I change the top menu icon?

### DISCUSSION THREAD

**Discussion by psaenzdetejada** [nunocoracao/blowfish#1147] [Q&A] [answered]
Hi all,

I was checking the https://blowfish.page/ website, and notice the icon on the top left of the menu bar. But when I built my website, that image/icon doesn't appear.

![image](https://github.com/nunocoracao/blowfish/assets/99508682/21e3b059-c416-452b-aa61-4ea0ba22c6f8)

Any idea how can I fix it and update the image? I managed to get the favicon and the homepage image correctly, but can't find how to show and update that concrete one.

Thanks,
Pablo

**Comment by gibru** [comment, marked as answer]
Hi Pablo,

Have you checked whether the logo under params of your language file is uncommented? (config/_default/languages.en.toml)
You'll find it under

[params]
...
logo = "img/blowfish_logo_transparent.png"

cheers,
gibru

**Reply by psaenzdetejada** [reply]
Nice! Thank you very much. I missed that param :)
